### DISCUSSION TITLE
ReactCountryFlag inside Select component

### DISCUSSION THREAD

**Discussion by belkocik** [chakra-ui/chakra-ui#5352] [Q&A]
Hello, when I put ` <ReactCountryFlag countryCode="US" svg />` inside option there is an` [object Object]`  text next to EN:
![image](https://user-images.githubusercontent.com/62507500/148686041-fd5f95f9-ddd8-4ffc-a266-99394f6df0ed.png)
What I got to do to display a component from ReactCountryFlag library inside Option?

```js
            <Select
              onChange={changeLanguage}
              defaultValue={locale}
              width="74px"
              display={{ base: 'none', md: 'inline-block' }}
              variant="outline"
              cursor="pointer"
              ml="30px"
            >
              <option value="en">EN  <ReactCountryFlag countryCode="US" svg /></option>
              <option value="pl">PL</option>
            </Select>
```
I'd like to display a flag next to this select button:  Would you please tell me the best way to do it?
edit: tried just import an image inside <option> and haven't worked, the same result: [object Object]
![image](https://user-images.githubusercontent.com/62507500/148686225-3261af7c-5d17-4fb0-8946-046bb014f4ca.png)

**Comment by crissmoldovan** [comment]
the problem here is you are trying to render a react node inside a native `option`.

could you please try to use: [chakra-react-select](https://github.com/csandman/chakra-react-select)  with a custom component for rendering options

(i will try to give you an example)

**Reply by belkocik** [reply]
Yes, would be nice if you share me a snippet. I will check how to use chakra-react-select tomorrow. Anyway, thank you for your help in advance. :-)

**Reply by belkocik** [reply]
I'm hopeful :D

**Reply by tuxin** [reply]
Hi, 

You can share you code for image in option ?
